crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -72,12 +76,24 @@ impl ActionHandler for BridgeLockAction {
             .await
             .context("failed to get transfer base fee")?;
 
+        let transaction_hash = state
+            .get_current_source()
+            .ok_or(anyhow!("expected current source to be `Some`"))?
```

Comment:
the transaction source should always be set, right? if so, you can `expect("current transaction source must be set")`